shell/src/test/java/alluxio/cli/fsadmin/command/SummaryCommandTest.java

```diff
@@ -0,0 +1,193 @@
+  @After
+  public void cleanDependencies() throws IOException {
+    mMetaMasterClient.close();
+    mBlockMasterClient.close();
+    mPrintStream.close();
+    mFile.delete();
+  }
+
+  @Test
+  public void callSummaryWithDependencies() throws IOException {
+    SummaryCommand summaryCommand = new SummaryCommand(mMetaMasterClient,
+        mBlockMasterClient, mPrintStream);
+    summaryCommand.run();
+    mPrintStream.flush();
+    mPrintStream.close();
+
+    BufferedReader reader = new BufferedReader(new FileReader(mFile));
+    checkIfOutputValid(reader);
+  }
+
+  /**
+   * Checks if the output contains expected values.
+   *
+   * @param reader reader to read SummaryCommand output
+   */
+  private void checkIfOutputValid(BufferedReader reader) throws IOException {
+    StringBuilder outputBuilder = new StringBuilder();
+    String line;
+    while ((line = reader.readLine()) != null) {
+      outputBuilder.append(line);
+    }
+    String output = outputBuilder.toString();
+
+    Assert.assertThat(output, CoreMatchers.containsString(
+        "Master Address: " + mAddress));
+    Assert.assertThat(output, CoreMatchers.containsString(
+        "Web Port: " + mWebPort));
+    Assert.assertThat(output, CoreMatchers.containsString(
+        "Rpc Port: " + mRpcPort));
+    Assert.assertThat(output, CoreMatchers.containsString(
+        "Started: " + CommonUtils.convertMsToDate(mStartTimeMs)));
+    Assert.assertThat(output, CoreMatchers.containsString(
+        "Uptime: " + CommonUtils.convertMsToClockTime(mUpTimeMs)));
+    Assert.assertThat(output, CoreMatchers.containsString(
+        "Version: " + mVersion));
+    Assert.assertThat(output, CoreMatchers.containsString(
+        "Safe Mode: " + mSafeMode));
+
+    Assert.assertThat(output, CoreMatchers.containsString("Live Workers: " + mLiveWorkerNum));
+    Assert.assertThat(output, CoreMatchers.containsString("Lost Workers: " + mLostWorkerNum));
+    Assert.assertThat(output, CoreMatchers.containsString("Total Capacity: "
+        + FormatUtils.getSizeFromBytes(mCapacityBytes)));
+    Assert.assertThat(output, CoreMatchers.containsString("Total Capacity: "
+        + FormatUtils.getSizeFromBytes(mCapacityBytes)));
+    Assert.assertThat(output, CoreMatchers.containsString("Used Capacity: "
+        + FormatUtils.getSizeFromBytes(mUsedBytes)));
+    Assert.assertThat(output, CoreMatchers.containsString("Free Capacity: "
+        + FormatUtils.getSizeFromBytes(mFreeBytes)));
+    Assert.assertThat(output, CoreMatchers.containsString("Tier: MEM"
+        + "  Size: " + FormatUtils.getSizeFromBytes(mCapacityBytesOnTiers.get("MEM"))));
+    Assert.assertThat(output, CoreMatchers.containsString("Tier: MEM"
+        + "  Size: " + FormatUtils.getSizeFromBytes(mUsedBytesOnTiers.get("MEM"))));
+  }
+
+  /**
+   * Sets the info values so that we can do explicitly check.
+   */
+  private void setInfoValues() {
+    mAddress = NetworkAddressUtils.getConnectAddress(ServiceType.MASTER_RPC).toString();
```

Comment:
I would specify explicit fake values for all of these fields so that we know exactly what they will be. e.g. 
```java
mAddress = "testaddress";
mWebPort = 4274;
mRpcPort = 9385;
mVersion = "test-version";
```
this way we can write a string representing the exact expected output

Reply:
changed to private static final constants with random values.
```java
  private static final String ADDRESS = "testAddress";
  private static final int WEB_PORT = 1231;
  private static final int RPC_PORT = 8462;
  private static final long START_TIME_MS = 213124234312231L;
  private static final long UPTIME_MS = 12412412312L;
  private static final String VERSION = "testVersion";
  private static final boolean SAFE_MODE = false;
  private static final int LIVE_WORKER_NUM = 12;
  private static final int LOST_WORKER_NUM = 4;
  private static final long CAPACITY_BYTES = 1341353L;
  private static final long USED_BYTES = 62434L;
  private static final long FREE_BYTES = 1278919L;
```
Thanks